app/stac_api/pgtriggers.py

```diff
@@ -262,28 +249,68 @@ class UpdateCollectionUpdateIntervalTrigger(pgtrigger.Trigger):
         RETURN asset_instance;
         '''
 
+    class UpdateDeleteCounterTrigger(pgtrigger.Trigger):
+        when = pgtrigger.After
+        declare = [('asset_instance', 'stac_api_collectionasset%ROWTYPE'),
+                   ('related_collection_id', 'INT')]
+        func = '''
+        asset_instance = OLD;
+
+        related_collection_id = asset_instance.collection_id;
+
+        UPDATE stac_api_projepsgcount
+        SET count = count-1
```

Comment:
same here, why not remove when `count=0`?

Reply:
Yes, I forgot to look at that. I changed it so now the entry should be removed if count is 0. Thanks!